src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/EstimatedDocumentCountCommand.java

```diff
@@ -0,0 +1,12 @@
+package io.stargate.sgv2.jsonapi.api.model.command.impl;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.stargate.sgv2.jsonapi.api.model.command.NoOptionsCommand;
+import io.stargate.sgv2.jsonapi.api.model.command.ReadCommand;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(
+    description =
+        "Command that returns estimated count of documents in a collection based on the collection.")
```

Comment:
"based on the collection." reads a bit strange - maybe "based on the collection metadata." or something similar?

Reply:
Ok this was missed in code review; definitely accidentally weird. :)